docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.9"
+
+services:
+  backend:
+    build: .
+    depends_on:
+      - db
+    ports:
+      - 8080:8080
+    container_name: try-cb-api
+
+  frontend:
+    build: "https://github.com/couchbaselabs/try-cb-frontend-v2.git#7.0"
+    ports:
+      - 8081:8081
+    container_name: try-cb-fe
+    entrypoint: ["wait-for-it", "backend:8080", "--timeout=400", "--strict", "--", "npm", "run", "serve"]
+    depends_on:
+      - backend
+
+  db:
+    image: build-docker.couchbase.com:443/couchbase/server-internal:7.0.0-5247
```

Comment:
I guess for testing purposes we'd use the internal image, but probably we can change this to the sandbox image when 7.0 GA is publicly available?
 
 https://github.com/couchbase/server-sandbox/tree/7.0.0-beta

Reply:
Makes sense however it won't actually work with the beta version anyway.